tests/python/sparsetir/test_tir_sparse_correctness.py

```diff
@@ -113,7 +115,40 @@ def sddmm_tir(a: T.handle, b: T.handle, c: T.handle, indptr: T.handle, indices:
             with T.init():
                 C_data[vij] = 0.
             C_data[vij] = C_data[vij] + \
-                A[T.lower_bound(C_indptr.data, vij, 0, M + 1) * K + vk] * B[C_indices[vij] * K + vk]
+                A[(T.upper_bound(C_indptr.data, vij, 0, M + 1) - 1) * K + vk] * B[C_indices[vij] * K + vk]
+
+
+@T.prim_func
+def bmm_tir(a: T.handle, b: T.handle, c: T.handle,
+            indptr_i: T.handle, indptr_j: T.handle, indptr_k: T.handle,
+            indptr_ij: T.handle, indptr_jk: T.handle, indptr_ik: T.handle,
+            BATCH: T.int32,
+            NNZ_IJ: T.int32, NNZ_JK: T.int32, NNZ_IK: T.int32) -> None:
+    T.func_attr({"global_symbol": "main", "tir.noalias": True})
+    A = T.match_buffer(a, (NNZ_IJ,), "float32")
+    B = T.match_buffer(b, (NNZ_JK,), "float32")
+    C = T.match_buffer(c, (NNZ_IK,), "float32")
+    indptr_I = T.match_buffer(indptr_i, (BATCH + 1,), "int32")
+    indptr_J = T.match_buffer(indptr_j, (BATCH + 1,), "int32")
+    indptr_K = T.match_buffer(indptr_k, (BATCH + 1,), "int32")
+    indptr_IJ = T.match_buffer(indptr_ij, (BATCH + 1,), "int32")
+    indptr_JK = T.match_buffer(indptr_jk, (BATCH + 1,), "int32")
+    indptr_IK = T.match_buffer(indptr_ik, (BATCH + 1,), "int32")
+    for b in T.grid(BATCH):
+        with T.block("bmm_outer"):
+            T.block_attr({"sparse": True})
+            vb = T.axis.S(BATCH, b)
+            with T.init():
```

Comment:
@MasterJH5574  Note that w/o these two lines we cannot identify `bmm_outer` as a reduction block, any ideas on what should we do to this these?